book/3 - Configuring world.ini/4 - Storage.html

```diff
@@ -1,5 +1,5 @@
 <p>
-	Cuberite supports several storage schemas. Note that Cuberite tries to load chunks from all storage schemas, no matter what this setting is set to. But only the storage set here will be used for saving chunks. This means that a world can be primed with MCA files from a Notchian server and still use the Compact schema for saving.
+	This section specifies if Cuberite should save world chunks or not, as well as the compression level for world files.
```

Comment:
No, it's fine. I apologize. I just never noticed we have a compression level setting :P